blog/20121016mate-package-repository-for-ubuntu-quantal.md

```diff
@@ -8,8 +8,8 @@
 .. author: Steve Zesch
 -->
 
-An [Ubuntu](http://www.ubuntu.org) package repository for Ubuntu 12.10 Quantal Quetzal
-is now available for our 1.4 release. As usual, follow [these instructions](http://wiki.mate-desktop.org/download/).
+An [Ubuntu](https://www.ubuntu.org) package repository for Ubuntu 12.10 Quantal Quetzal
```

Comment:
http://www.ubuntu.org seems to be a dead page

https://www.ubuntu.org doesn't work

Reply:
i will use https://ubuntu.com/ .

Reply:
Done